lib/screen/login_page.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:flutter/material.dart';
+
+import '../utils/constants.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: CustomPaint(
+        painter: Bluepainter(),
+        child: Container(
+          width: double.infinity,
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              // Sochem icon
+              Container(
+                width: MediaQuery.of(context).size.width * 0.5,
+                height: MediaQuery.of(context).size.height * 0.6,
```

Comment:
Just like this there are many usages of MediaQuery in the file. Instead of retrieving screen height and width again, just store them in global variables and use wherever required.